libraries/Microsoft.Bot.Builder.Dialogs.Rules/Steps/HttpRequest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Rules.Steps
+{
+    public class HttpRequest : DialogCommand
```

Comment:
If we support only json, can we validate the body and if the user provides no json then tell them that we only support jkson payloads?

Reply:
> If we support only json, can we validate the body and if the user provides no json then tell them that we only support jkson payloads?
If user pass other types the bot will throw an exception while loading and won't start at the beginning. The json structure is predefined by the developers rather than users.